src/SwedbankPay.Sdk.Infrastructure/ProblemResponse.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace SwedbankPay.Sdk
+{
+    public class ProblemResponse : IProblemResponse
+    {
+        public ProblemResponse()
+        {
+        }
```

Comment:
To support it in JsonSerialization. I can make it into a dto an map it back to this class, but I thought since this is in the Infrastructure project I could do a shortcut.